### DISCUSSION TITLE
When going to take Sega's model1,model2,model3 emulators all together for the ultimate slam dunk!

### DISCUSSION THREAD

**Discussion by cjhalo** [flyinghead/flycast#1185] [Ideas]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by flyinghead** [comment]
These platforms have nothing in common with Dreamcast/Naomi and emulators already exist for them. So it won't happen.

**Comment by cinnamonmatexfce** [comment]
In case you're not aware of, there's [Supermodel](https://github.com/trzy/Supermodel), which is a SEGA Model3 emulator! ✌️
